yarn-project/foundation/src/crypto/pedersen/pedersen.wasm.ts

```diff
@@ -21,16 +22,17 @@ export function pedersenCommit(input: Buffer[]) {
  * Create a pedersen hash (field) from an array of input fields.
  * Left pads any inputs less than 32 bytes.
  */
-export function pedersenHash(input: Buffer[], index = 0): Fr {
-  if (!input.every(i => i.length <= 32)) {
+export function pedersenHash(input: Bufferable[], index = 0): Fr {
```

Comment:
it's very common that we pass an array of fields to this function so it makes sense to allow passing of Bufferable here.